MsixCore/msixmgr/msixmgr_tracelogging.h

```diff
@@ -0,0 +1,168 @@
+#pragma once
+
+#include<windows.h>
+#include <TraceLoggingProvider.h>
+#include "msixmgrTelemetry.hpp"
+#include <string>
+#include "..\msixmgrLib\GeneralUtil.hpp"
+using namespace MsixCoreLib;
+
+TRACELOGGING_DECLARE_PROVIDER(g_MsixMgrTelemetryProvider);
+
+namespace msixmgr
+{
+    inline void TraceLogSession(const wchar_t* WorkflowID, const wchar_t* SourceApplicationID, const wchar_t* CorrelationID)
```

Comment:
I think casing of the parameters should be workflowId and not WorkflowID as this has been the standard getting followed in existing file UnpackProvider.cpp

Reply:
Done